web_pwa_oca/static/src/js/worker/base/tools.js

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2020 Tecnativa - Alexandre D. Díaz
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+ "use strict";
```

Comment:
Using this outside of a function can have side effects if the file is minimized and bundled with other ones. Such as how odoo assets do. Is that the case? If so, better declare a namespace (either with `odoo.define` or just `function(){...}`.

Reply:
These scripts are not minified, are injected directly by the browser.

See: https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts